cmd/plugins.go

```diff
@@ -0,0 +1,501 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/go-flutter-desktop/hover/internal/fileutils"
+	"github.com/go-flutter-desktop/hover/internal/log"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	yaml "gopkg.in/yaml.v2"
+)
+
+const standaloneImplementationListAPI = "https://raw.githubusercontent.com/go-flutter-desktop/plugins/master/list.json"
+
+var (
+	listAllPluginDependencies bool
+	cleanPurge                bool
+	dryRun                    bool
+	reImport                  bool
+)
+
+func init() {
+	pluginTidyCmd.Flags().BoolVar(&cleanPurge, "purge", false, "Remove all go platform plugins imports from the project.")
+	pluginListCmd.Flags().BoolVarP(&listAllPluginDependencies, "all", "a", false, "List all platform plugins dependencies, even the one have no go-flutter support")
+	pluginGetCmd.Flags().BoolVar(&reImport, "force", false, "Re-import already imported plugins.")
+
+	pluginCmd.PersistentFlags().BoolVarP(&dryRun, "dry-run", "n", false, "Perform a trial run with no changes made.")
+
+	pluginCmd.AddCommand(pluginListCmd)
+	pluginCmd.AddCommand(pluginGetCmd)
+	pluginCmd.AddCommand(pluginTidyCmd)
+	rootCmd.AddCommand(pluginCmd)
+}
+
+var pluginCmd = &cobra.Command{
```

Comment:
@GeertJohan 
`hover plugin get`, `hover plugin list`..

It feels a little bit like we're letting hover work on a plugin that was created with `init-plugin`...

A project can have multiple plugins. So should this subcommand maybe be plural?

`hover plugins get`, `hover plugins list`?